### PR TITLE
Implemented a UI test framework

### PR DESCRIPTION
**Pull request by bparish628**
I decided to just use Karma/Jasmine.

### CODE REVIEW

**Review by jakeharding — COMMENTED** [review]
Excellent.  Could you include the doc strings in your source files in this pr?

**Comment by bparish628** [comment]
I actually had it in another branch, but I moved it into here! @jakeharding

File: repo_health/index/static/src/components/search/search.spec.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+describe('Search', () => {
```
Comment (jakeharding) [inline repo_health/index/static/src/components/search/search.spec.js:13]: Let's add the doc strings to test files also.

**Review by jakeharding — APPROVED** [review]
Sweet. Thanks.

**Comment by bparish628** [comment]
:+1:

**Pull request merged**